pkg/configuration/roles/tendermint/files/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG tm_version=0.19.2
```

Comment:
Shouldn't the version be updated to `0.19.3` ?

Reply:
yes